docs/process-for-changing-ion-schema.md

```diff
@@ -0,0 +1,177 @@
+
+## 3. An Ion Schema Version RFC will adopt the feature
+
+*Ion Schema Version RFCs* are meta-RFCs that describe a new version of Ion Schema. They are PRs created by the **maintainers**, not by community members proposing a new feature. Version RFCs include a `.md` file that contains:
+
+* A list of the feature RFCs to be included in the new version, including for each a brief summary and a link to the accepted RFC for that feature. Links to other documentation may also be included.
+* The list of all accepted feature RFCs that are *not* included in the new version, with a reason why it is not included.
+* If it is a new major version, a definition of the compatibility and import requirements between this version and the prior major version, as required by [Ion Schema Language Versions](https://quip-amazon.com/WxDyAMCsxccV).
+
+For an example, please see [the Ion Schema 2.0 RFC](https://github.com/amazon-ion/ion-schema/blob/208165adb10c889949252e7ccd926862bfe60019/rfcs/ion_schema_2_0/ion_schema_2_0.md) ([PR](https://github.com/amazon-ion/ion-schema/pull/69)).
+
+Ion Schema versions are not released on a fixed schedule. The creation of a version RFC is a required step on the path to release but is not an indication of any particular timeline.
+
+Ion Schema version RFCs are not finalized until they are merged; feature RFCs may be moved between them or removed altogether. Removal does not revoke a previously accepted feature RFC; it only removes it from that version.
+
+## 4. A new Ion Schema spec version will be released
+
+When developer bandwidth is available, the next Ion Schema version RFC will be merged. At this point, **contributors** (in all likelihood, mostly just the **maintainers**) will write a new version of the Ion Schema specification that incorporates the changes in the Ion Schema Version RFC. Designing the exact format of the new specification document, including how individual features’ supported versions will be communicated, is outside of the scope of this document.
+
+Once the new version of the spec has been merged (via one or more pull requests), Ion Schema libraries can be updated to add support for the features that were included in the new version.
+
+As before, announcements will be added to the Ion home and news pages.
+
+# Internal steps for considering a proposal
+
+*This section is addressed to the Ion Schema ** **maintainers**.* *It is included here in order to be transparent about the approval process.*
+
+## Requesting an RFC
```

Comment:
Should we consider a Service Level Objective (SLO), e.g. The "2 and 5 Promise" for interviewing?